master/internal/api_notebook.go

```diff
@@ -0,0 +1,29 @@
+package internal
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/determined-ai/determined/proto/pkg/apiv1"
+)
+
+func (a *apiServer) GetNotebooks(
+	_ context.Context, req *apiv1.GetNotebooksRequest,
+) (resp *apiv1.GetNotebooksResponse, err error) {
+	err = a.actorRequest("/notebooks", req, &resp)
+	if err != nil {
+		return nil, err
+	}
+	a.sort(resp.Notebooks, req.OrderBy, req.SortBy, apiv1.GetNotebooksRequest_SORT_BY_ID)
+	return resp, a.paginate(&resp.Pagination, &resp.Notebooks, req.Offset, req.Limit)
+}
+
+func (a *apiServer) GetNotebook(
+	_ context.Context, req *apiv1.GetNotebookRequest) (resp *apiv1.GetNotebookResponse, err error) {
```

Comment:
On `_ context.Context` here and everywhere else: in a perfect world we would pass this along right (doesn't it contain time info, etc)? should we add a TODO maybe?

Reply:
The context here contains the gRPC metadata which is useful for certain things (like getting user information). I think it is worth only passing along if it is needed downstream.